src/Shell/Task/ServerTask.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace RabbitMQ\Shell\Task;
+
+use RabbitMQ\CakephpRabbitMQ;
+
```

Comment:
plus... not sure what's the point to have a shell without methods and a task with one method. shell's main method should do this